packages/site-demo/content/product/components/expansion-panel/react/examples.tsx

```diff
@@ -33,25 +21,22 @@ export const App = () => {
                 toggleButtonProps={{ label: 'Toggle' }}
             >
                 <header>
-                    <FlexBox
+                    <GenericBlock
                         className="lumx-spacing-margin-left-regular"
-                        orientation={Orientation.horizontal}
-                        hAlign={Alignment.center}
+                        hAlign="center"
+                        figure={<Thumbnail alt="" image="/demo-assets/square1.jpg" size="m" variant="rounded" />}
```

Comment:
The design tokens are no longer relevant (Size, "Variant", Typography etc.)? Maybe, we should use them in our examples files to encourage their use?